gfx/portraits/portrait_modifiers/01_headgear.txt

```diff
@@ -35,6 +35,14 @@ headgear = {
 				prefer_no_lowborn_headgear = yes
 			}
 			
+			# Warcraft
+			modifier = {
+				add = 1000
+				# No helmet during war
+				prefer_no_headgear_culture_trigger = yes
+				portrait_wear_armor_trigger = yes
```

Comment:
This is because without `portrait_wear_armor_trigger = yes` the Wildhammer will not wear headgear even during peacetime.

Reply:
> This is because without `portrait_wear_armor_trigger = yes` the Wildhammer will not wear headgear even during peacetime, including crowns.

The Wildhammer have `no_headgear_culture_gfx` kinda saying they don't wear headgear at all.

Reply:
Explain what you want for every culture because your comments made me confused.

Reply:
> Duke tier and lower Wildhammer characters do not wear helmets during war.

From your changelog, it looks like you want to edit `prefer_no_lowborn_headgear` and `portrait_wear_helmet_trigger`.

Reply:
Resolved. Intention was to let Wildhammer wear headgear during peacetime but no helmets during war.